pg2kinesis/formatter.py

```diff
@@ -83,15 +90,126 @@ def produce_formatted_message(self, change):
 
 class CSVFormatter(Formatter):
     def produce_formatted_message(self, change):
-        msg = Message(change=change, fmt_msg='{},{},{},{},{},{}'.format(CSVFormatter.VERSION,
-                                                                         CSVFormatter.TYPE, *change))
+        msg = Message(
+            change=change,
+            fmt_msg='{},{},{},{},{},{}'.format(CSVFormatter.VERSION, CSVFormatter.TYPE, *change),
+            is_bulk=False,
+        )
         return msg
 
 
 class CSVPayloadFormatter(Formatter):
     def produce_formatted_message(self, change):
-        msg = Message(change=change, fmt_msg='{},{},{}'.format(CSVFormatter.VERSION,
-                                                                CSVFormatter.TYPE, json.dumps(change.__dict__)))
+        msg = Message(
+            change=change,
+            fmt_msg='{},{},{}'.format(CSVFormatter.VERSION, CSVFormatter.TYPE, json.dumps(change.__dict__)),
+            is_bulk=False,
+        )
+        return msg
+
+
+class JSONPayloadFormatter(Formatter):
```

Comment:
You can get rid of most of this if we make this always return a list of columns, even if it just the primary column.